app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -326,7 +326,7 @@ export default {
       <a href="/play/ladder/blazing-battle" class="btn btn-large btn-primary btn-moon" style="padding: 20px 100px;">Play Blazing Battle Multiplayer Arena</a>
     </div>
 
-    <section class="row flex-row">
+    <section class="row flex-row free-to-get-start">
```

Comment:
👏  This is awesome! See [other comment](https://github.com/codecombat/codecombat/pull/6104/files#r551602515). The background image shown by this class needs to only appear selectively.

Using Vue object syntax works: https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax

Reply:
Added another class `free-to-get-start-bg` to display background conditionally.